test/index.test.js

```diff
@@ -205,6 +205,112 @@ describe('Dynastar - index.js', function () {
     });
   });
 
+  describe('key builders', function () {
+    let myModel, myHashKey, myRangeKey, myWrapped;
+
+    before(function () {
+      myHashKey = 'key';
+      myRangeKey = 'what';
+
+      myModel = dynamo.define('test2', {
+        hashKey: myHashKey,
+        rangeKey: myRangeKey,
+        schema: {
+          key: Joi.string(),
+          hello: Joi.string(),
+          what: Joi.string(),
+          other: Joi.string().allow(null)
+        }
+      });
+    });
+
+    afterEach(function (done) {
+      if (myWrapped) {
+        myWrapped.dropTables(done);
+      } else {
+        done();
+      }
+    });
```

Comment:
Here. this one should pass the linter
```suggestion
      if (myWrapped) {
        return void myWrapped.dropTables(done);
      }
      done();
    });
```